manifest/schema1/manifest_test.go

```diff
@@ -79,7 +86,7 @@ func genEnv(t *testing.T) *testEnv {
 
 	name, tag := "foo/bar", "test"
 
-	m := Manifest{
+	invalid := Manifest{
```

Comment:
It's here:

https://github.com/docker/distribution/pull/1156/files#diff-28cb844cece467ec95e88df11956c665R73